apps/api-gateway/src/dependency-integrity.spec.ts

```
@@ -0,0 +1,101 @@
+    catalog[name] = range;
+  }
+  return catalog;
+}
+
+function declaredRange(pkg: string, dependencies: Record<string, string>, catalog: Catalog): string {
+  const declared = dependencies[pkg];
+  return 'catalog:' === declared ? (catalog[pkg] ?? declared) : declared;
+}
+
+const toTuple = (value: string): number[] =>
+  value
```

ESLint

[error] Expected no linebreak before this expression.

(implicit-arrow-linebreak)